vep/__init__.py

```diff
@@ -61,11 +76,14 @@ def _power_on_self_test(self):
             )
             self.python = python_real_path
 
-    @staticmethod
-    def _get_setup_attribute(attribute_name):
-        pycmd = sh.Command('python')
-        if os.path.isfile("setup.py"):
-            return pycmd('setup.py', "--%s" % attribute_name).strip()
+    def get_setup_option(self, option):
+        pycmd = sh.Command(os.path.join(self.target, 'bin', 'python'))
+        if self.setup_options[option] is None:
+            if getattr(self.args, self.setup_option_names[option]) is not None:
+                self.setup_options[option] = self.args.getattr[self.setup_option_names[option]]
+            else:
+                self.setup_options[option] = pycmd('setup.py', "--%s" % option).strip()
+        return self.setup_options[option]
```

Comment:
Did you ever figure out why the linter was whining? If not, it probably merits a comment.